README.md

```diff
@@ -23,6 +24,19 @@ Edit `plugin.json` with your `id`, `name`, and `description`:
 }
 ```
 
+2. `go.mod` with your Go module path:
```

Comment:
Should be encourage developers to use there repository path as module path?

Reply:
How about:
```suggestion
2. `go.mod` with your Go module path, following the `<hosting-site>/<repository>/<module>` convention:
```
?

The `com.example.my-plugin` of the README is hosting-site agnostic, but as the template contains `github.com/mattermost/mattermost-plugin-starter-template`, it should be enough of a guide for Github&non-Github users.

Reply:
I like the proposal :+1: Given that this template is on GitHub I'm would like to see https://github.com/mattermost/mattermost-plugin-starter-template getting used as an example.